### ISSUE TITLE
Standard Notes recent conflict regression - conflicted tags, and note conflict appears during typing

### ISSUE THREAD

**Issue by k-stanislawek** [standardnotes/forum#3307]
**Describe the bug**

1. Recently, last week or so, conflcits are showing up more often, and for the first time, there are conflicts of Tags. I currently have 4 copies of the same tag, with slightly different number of notes (282, 281 279, 289).

2. Moreover, today the note has got a conflict in the middle of writing. The word I was typing into the browser (Macos chrome) got deleted on my screen, and I was notified of a conflict, between the versions without that word & with that word. No other Standard Notes window was opened at the time.

3. Now, when I type the note, it can delete my last 3 seconds & send the cursor to the end of the note in the middle of the sentence. No conflict this time, these few seconds of typed text seem permanently lost. It's a blocker for me, I'll temporarily use an alternative.

**To Reproduce**

No idea

**Expected behavior**

No conflicts at all, or at least, two copies of the same note - not two copies of the same tag, with 300 notes each, it makes it incredibly hard to merge back

**Screenshots**

I can't post a screenshot because notes have sensitive content. I'm hoping that I'm not alone.

**Desktop (please complete the following information):**
 - Browser: Updated Chrome on different devices. The note conflict error appeared on MacOS Chrome, Version 117.0.5938.149. I'm also using it from other browsers.

**Smartphone (please complete the following information):**
- Samsung Galaxy A52s, Android 13

**Comment by effieeee** [comment]
Hi @k-stanislawek, I'm very sorry to hear about this. You can first delete the duplicated tags, keeping only the one with the most number of notes linked to it. Once this is done, please try signing out of your account on all devices first, then sign back into your account on each one, and try to check if the issue will recur afterwards

**Comment by rokfish2** [comment]
I was looking at Standard Notes to see if it is an alternative to my evernote and went online to their demo page.
The same thing seems to be happening on that one too.
https://standardnotes.com/demo

Its when I click on any of the tags that have subtags and they drop down, I see conflicted copies.


![2023-10-28_092952](https://github.com/standardnotes/forum/assets/149214767/1cea1ff3-92bc-4ec0-ada8-654c014cf949)

**Comment by effieeee** [comment]
Hi @rokfish2, the conflict issues on the demo app are a separate issue, which is also being looked into 🙏

**Comment by ekopunov** [comment]
I'm also getting heaps of conflicted copies of notes - literally any time I type anything I'm getting one or two conflicted copies - this is Standard Notes premium web-version, same thing happening on the app, latest version on Ubuntu 21.

**Comment by tiguchi** [comment]
Chiming in here to confirm that I also have this issue and it's getting worse.

I already signed out of my workspace, signed back in again, as suggested by @effieeee. I use StandardNotes only on a single device, so there is only one session at a time. I still run into data loss issues while typing. Entire sentences get wiped out when that "conflict" error appears.

I do not really need online-backups while typing a document. It would be more than sufficient when a snapshot is taken after I switch to a different note, or when I close the application.

I would even prefer a manual sync feature over the current automatic sync feature.

**Comment by rlindner81** [comment]
I also have the issue with a _massive_ increase in conflicts when I'm just writing my notes on one device. It seems to be related to the automatic syncing. It works ok for a bit, but when I'm in my flow and writing maybe half a page, I consistently get at least 2 conflicts. This is completely bonkers, because it's just one device, one text. In past versions of SN I never had this problem.

**Comment by ekopunov** [comment]
I agree - it's also very distracting to see the sync firing every time I type a character, especially when it fails half the time. I would welcome an option to choose a more relaxed algorithm for the syncs - say every 30s plus when I click out of the note. The way it is it's doing more harm than good, especially since everything is supposed to be stored locally anyway - why have this paranoid sync-every-character-immediately policy then?

**Comment by effieeee** [comment]
Hi all, just letting you know that the recent incidents with syncing and note conflicts may not be related to the original issue posted here. However, the team is working hard to investigate the issue on the servers and arrive at a fix. The investigation is still in its early stages, and it looks like it will need more thorough digging and troubleshooting, but we're doing our best to resolve this as soon as possible, and I hope you will bear with us as we continue to look into this 🙏

**Comment by rlindner81** [comment]
Small update, there seem to be changes happening with the sync on server-side, but they make the problem much worse for me. It used to be that I can write half a page and it would just _show_ me that sync issues had occurred. That is a little annoying, but not more. Now the problem is worse. It will actively _remove_ text while I'm writing it... usually about 3--4 words. Hope someone finds a fix soon--it used to work--or the users get an option to turn auto syncing off.

**Comment by effieeee** [comment]
Hi, the recent syncing issues have been addressed recently, so you shouldn't be seeing constant conflicts on your notes at this point 🙏 If you continue to experience the same issues, you can send us a message at help@standardnotes.com using your account email so that we can try to assist you further with it!

**Comment by ekopunov** [comment]
> Hi, the recent syncing issues have been addressed recently, so you shouldn't be seeing constant conflicts on your notes at this point 🙏 If you continue to experience the same issues, you can send us a message at [help@standardnotes.com](mailto:help@standardnotes.com) using your account email so that we can try to assist you further with it!

Thanks, I can confirm I haven’t seen the conflicts yesterday and today.
